docs/releasing/publishing.md

```diff
@@ -67,8 +67,9 @@
 npm logout
 ```
 
-17. Send a message to our users in both the X-GOV and GDS #govuk-design-system slack channels that indicates
-there's a new release with a short summary.
+17. Post a short summary of the release in the cross-government and GDS #govuk-design-system Slack channels. For example:
+
+    🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. [https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
```

Comment:
Could we also add something about acknowledging the help we got from different contributors.

Reply:
Ah yes, sorry, I forgot about that - will add! Thanks Hanna.

Reply:
Added in f92358b! Just in the Slack comms section for now so we keep this update in scope.